mc-server-wrapper-lib/src/parse.rs

```diff
@@ -64,14 +64,21 @@ impl ConsoleMsgSpecific {
             ConsoleMsgSpecific::PlayerAuth { name, uuid }
         } else if console_msg.msg_type == ConsoleMsgType::Info
             && (console_msg.thread_name.starts_with("Async Chat Thread")
-                || console_msg.msg.starts_with('<') && console_msg.thread_name == "Server thread")
+                || (console_msg.msg.starts_with('<')
+                    || console_msg.msg.starts_with("[Not Secure]"))
+                    && console_msg.thread_name == "Server thread")
```

Comment:
this was made by fmt, btw. kinda ridiculous. 🤢

Reply:
Haha no worries! The formatter is queen 👸

Also the code in this file is kinda bad, I threw it together years ago and it's served its purpose ever since. In an ideal world all these long conditions wouldn't even exist.

Reply:
Haha no worries! The formatter is queen 👸

Also the code in this file is kinda bad, I threw it together years ago and it's served its purpose ever since. In an ideal world all these long conditions wouldn't even exist.